harness/Makefile

```diff
@@ -13,10 +13,10 @@ fmt:
 
 .PHONY: check
 check: gen
-	isort --check-only
+	isort --check-only --skip client
 	black . --check
 	flake8
-	mypy .
+	mypy . --show-error-context --show-error-codes
```

Comment:
is this change still necessary?  It seems like a debug setting, not something generally useful to developers.

Reply:
the error codes help finding out when some check fails